src/Compilers/Core/Portable/Compilation/CompilationOptions.cs

```diff
@@ -339,6 +339,11 @@ internal bool CanReuseCompilationReferenceManager(CompilationOptions other)
                 && object.Equals(this.AssemblyIdentityComparer, other.AssemblyIdentityComparer);
         }
 
+        /// <summary>
+        /// Gets the source language ("C#" or "Visual Basic").
+        /// </summary>
+        public abstract string Language { get; }
```

Comment:
We have SyntaxNode.Language, ISymbol.Language and Project.Language.  I don't believe we really use 'LanguageName' in our public APIs.  I think this should probably match those other APIs.  Wht do you think Neal?